src/components/Table/GridTable.tsx

```diff
@@ -232,11 +232,15 @@ export function GridTable<R extends Kinded, X extends Only<GridTableXss, X> = {}
       tableState.columns
         .filter((c) => tableState.visibleColumnIds.includes(c.id))
         .flatMap((c) =>
-          c.expandColumns && tableState.expandedColumnIds.includes(c.id) ? [c, ...c.expandColumns] : [c],
+          c.expandColumns && Array.isArray(c.expandColumns) && tableState.expandedColumnIds.includes(c.id)
```

Comment:
Question: Can the `Array.isArray` be removed now that `expandedWidth` is a different property?

Reply:
Yup, definitely. Thanks for catching this one.